specs/interop/messaging.md

```diff
@@ -162,6 +165,34 @@ The expiry invariant invalidates inclusion of any executing message with
 - `executing_block` is the block where the executing message was included in.
 - `EXPIRY_TIME = 180 * 24 * 60 * 60 = 15552000` seconds, i.e. 180 days.
 
+## Special Messages
+
+### `logIndex` wildcard
+
+[wildcard]: #logindex-wildcard
+
+The author of an executing-transaction may defer the specification of an exact `logIndex` to the block-builder.
+Deferring the `logIndex` enables the executing side to request a message to be initiated.
+The `logIndex` value of `0x01000000000000000000000000000000000000000000000000000000000000` functions as wildcard.
+
+The block-builder is responsible for mapping the wildcard `logIndex` to a valid `logIndex` by appending
+it to a list of indices, declared statically in the input data of the last transaction of the block.
+
+This declaring transaction must have a `to` address of `0x42ff000000000000000000000000000000000000`.
```

Comment:
We should have an autogenerated specs page that documents all arbitrary addresses in the 0x42 namespace, or some CI check, to avoid collisions